.coafile

```diff
@@ -13,26 +13,26 @@ bears = SpaceConsistencyBear, QuotesBear
 language = python
 preferred_quotation = '
 
-[autopep8]
+[all.autopep8]
 bears = PEP8Bear, PycodestyleBear, PyDocStyleBear
 pydocstyle_ignore = D100, D101, D102, D103, D104, D107,
-  D200, D203, D205, D209, D212, D213, D400
+  D200, D203, D205, D209, D212, D213, D400, D105, D403,
```

Comment:
commit message says "improves the code style" , yet you are adding rules that are being ignored.